security/security_policy_compliance/timemachine.md

```diff
@@ -4,19 +4,22 @@ backup was taken. For this reason, encrypting these backups is important.
 
 ## First-Time Setup
 
-If you are not currently backing up the machine, follow the [Apple provided instructions](https://support.apple.com/en-us/HT201250) for setting up encrypted Time Machine backups to an external disk.
+System backups are not required, but are encouraged for he purpose of business
+continuity. If you choose not to maintain regulare system backups, it is
+_highly_ encouraged that you be able to bring your machine to a workable
+state quickly. For developers, managing your `dotfiles` is encouraged. Setup
+automation is also helpful. Here are a few examples:
 
-## Capturing Status
-Per the Sparkbox security policy, we regulary capture and record employees'
-status for each security policy item.
+  * [laptop script]
+  * [Cromwell's dotfiles & `mac` setup automation][cromwell dotfiles]
+  * [Adam's dotfiles][adam dotfiles]
 
-1. Choose Apple menu > System Preferences, then click Time Machine.
-2. Click Select Disk or Add or Remove Backup Disk (if you have multiple backup
-   disks).
-3. Capture a [screen shot][iOS Screenshot] which includes the status of all
-   Backup Disks.
+If you are not currently backing up the machine and would like to, follow the
+[Apple provided instructions](https://support.apple.com/en-us/HT201250) for setting up encrypted Time Machine backups to an external disk.
 
-Your disk should show _Encrypted_ under the name and after the disk size. If not, follow the instructions provided below.
+## Capturing Status
+We encourage you to use the [status check automation script][status check]
+to capture status for all Mac policies.
```

Comment:
If this is singular, I think it's missing an article ("capture the status") or if it's plural, it just needs pluralization ("capture the statuses").
```suggestion
to capture the status for all Mac policies.
```

Reply:
Resolved in b218534aaeb36941eeef26f2c86809bc8d07e246.